mlflow/projects/backend/local.py

```diff
@@ -94,9 +95,12 @@ def run(
         storage_dir = backend_config[PROJECT_STORAGE_DIR]
         build_image = backend_config[PROJECT_BUILD_IMAGE]
         docker_auth = backend_config[PROJECT_DOCKER_AUTH]
-        # Select an appropriate env manager for the project env type
         if env_manager is None:
-            env_manager = _env_type_to_env_manager(project.env_type)
+            # Check if the env manager is specified via an environment variable
+            env_manager = MLFLOW_ENV_MANAGER.get()
+            if env_manager is None:
+                # Select an appropriate env manager for the project env type
+                env_manager = _env_type_to_env_manager(project.env_type)
```

Comment:
To test, print the value of `env_manager` after it's set here (after the if-else block) and run:

```
# should print `virtualenv`
mlflow run -e hyperopt .
```

or

```
# should print `local`
mlflow run --env-manager=local -e hyperopt .
```